notebooks/data_analysis/pathfinder_items_other_columns.ipynb

```diff
@@ -0,0 +1,932 @@
+     },
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "items[pf2e.condition.notnull()]"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "All of items with `condition: True` have type *condition*\n",
+    "\n",
+    "*Not important*"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "**Flags**:\n",
+    "* core - *not important*\n",
+    "* pf2e:\n",
+    "    * linkedWeapon - maybe important if we are going to somehow count items of monster because e.g. Gancanagh have only one rapier (but a silver one)\n",
+    "    * condition - *not important*"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Summary\n",
+    "* _id - *not important*\n",
+    "* img - *not important*\n",
+    "* name - *not important*\n",
+    "* sort - **???**\n",
```

Comment:
Też nie wiem co to do końca, ale no jest to jakiegoś rodzaju id czegoś tam więc można olać